core/CHANGELOG.md

```diff
@@ -36,6 +36,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
```

Comment:
As we will probably release core right after this PR, can you already put the tag name above?